vcbuild.bat

```diff
@@ -469,6 +469,22 @@ if not defined noetw (
     copy /Y ..\src\res\node_etw_provider.man %TARGET_NAME%\ > nul
     if errorlevel 1 echo Cannot copy node_etw_provider.man && goto package_error
 )
+if defined dll (
+  copy /Y libnode.dll %TARGET_NAME%\ > nul
+  if errorlevel 1 echo Cannot copy libnode.dll && goto package_error
+
+  mkdir %TARGET_NAME%\Release > nul
+  copy /Y node.lib %TARGET_NAME%\Release\ > nul
+  if errorlevel 1 echo Cannot copy node.lib && goto package_error
+
+  copy /Y ..\common.gypi %TARGET_NAME%\ > nul
+  if errorlevel 1 echo Cannot copy common.gypi && goto package_error
```

Comment:
Also wondering what this is needed for

Reply:
As above, `node-gyp` tries to find `common.gypi` in the root of the `nodedir` if its not at `include/node/common.gypi`. I think this may be a remnant left over from before I figured out the HEADERS_ONLY install below. I'll try removing it